<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TelaTriagem</class>
 <widget class="QWidget" name="TelaTriagem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>398</width>
    <height>473</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Pressão</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>190</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Temperatura</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>260</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Peso</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>340</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Classificação</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtPressao">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>81</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="txtTemperatura">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>220</y>
     <width>81</width>
     <height>29</height>
    </rect>
   </property>
   <property name="suffix">
    <string>ºC</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="txtPeso">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>290</y>
     <width>81</width>
     <height>29</height>
    </rect>
   </property>
   <property name="suffix">
    <string>kg</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cbUrgencia">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>380</y>
     <width>82</width>
     <height>28</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Verde</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Amarelo</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Laranja</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Vermelho</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="btnSalvarTriagem">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>420</y>
     <width>101</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Salvar Triagem</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Registrar Triagem</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblPaciente">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Paciente:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtNomePaciente">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>60</y>
     <width>141</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnBuscarPaciente">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>60</y>
     <width>90</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Buscar</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
